config/subsidy-application-management/forms/urban-renewal/application-phase2/versions/20221218090000/form.ttl

```diff
@@ -869,7 +869,7 @@ ext:projectOnderdeelPg a form:PropertyGroup;
             form:validations
                               [ a                 form:MaxLength ;
                                 form:grouping     form:MatchEvery ;
-                                form:max          "500" ;
+                                form:max          "1000" ;
                                 sh:resultMessage  "Max. karakters overschreden." ;
                                 sh:path            lblodSubsidie:projectOnderdeelStadListingUnitAandeel
                               ];
```

Comment:
The character limit works as expected, but the counter doesn't show up (same for `Publieke partner` and `Private Partner`). Is it because it's in a table?

Reply:
Yea, some things are hidden when a field is displayed in a table to conserve space. It's honestly kind of weird to that users would input that many characters in a table. Especially when it's a normal input field.